dev/com.ibm.ws.concurrent.persistent/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -41,7 +41,7 @@ jndiName=JNDI name
 jndiName.desc=JNDI name.
 
 missedTaskThreshold=Missed task threshold for fail over
-missedTaskThreshold.desc=Amount of time beyond the expected start of a task execution to reserve for running it. Other members are prevented from running the task before the expiration of this interval. If the interval elapses without successful execution of the task, then the task execution is considered missed, enabling another member to attempt to run it. This enables fail over.
+missedTaskThreshold.desc=The amount of time after the expected start of a task execution that is reserved for running the task. Other members are prevented from running the task before the expiration of this interval. If the interval elapses without successful execution of the task or the task rolls back, then the task execution is considered missed, enabling another member to attempt to run it. Missed task threshold values within the supported range of 100 seconds to 9000 seconds (2.5 hours) enable failover.
```

Comment:
**suggested revision (removes "execution")

The amount of time after the expected start of a task  that is reserved for running the task. Other members are prevented from running the task before the expiration of this interval. If the interval elapses without successful completion of the task, or the task rolls back, then the task completion is considered missed, enabling another member to attempt to run it. Missed task threshold values within the supported range of 100 seconds to 9000 seconds (2.5 hours) enable failover.

Reply:
Updated here:
https://github.com/OpenLiberty/open-liberty/pull/10521/commits/f40a335f74e6e1ae21c568ba8343d1a5d4ffefc2